cognite/client/experimental/model_hosting/schedules.py

```diff
@@ -120,3 +147,17 @@ def delete_schedule(self, id: int) -> None:
         """
         url = "/analytics/models/schedules/{}".format(id)
         self._delete(url=url)
+
+    def get_log(self, id: int) -> ScheduleLog:
+        """Return schedule log by id.
+
```

Comment:
Add some more description here about the logs the user can expect to have returned. i.e. "Two logs, one for failed scheduled predictions and one for successful."